runners/samza/build.gradle

```diff
@@ -34,20 +34,6 @@ description = "Apache Beam :: Runners :: Samza"
 evaluationDependsOn(":sdks:java:core")
 
 repositories {
-  maven {
```

Comment:
IIRC, we need to update `runners/samza/job-server/build.gradle` as well

Reply:
also remove there as well, thanks for the reminder